lighthouse-core/audits/installable-manifest.js

```diff
@@ -5,38 +5,127 @@
  */
 'use strict';
 
-const MultiCheckAudit = require('./multi-check-audit.js');
-const ManifestValues = require('../computed/manifest-values.js');
+const Audit = require('./audit.js');
 const i18n = require('../lib/i18n/i18n.js');
+const ManifestValues = require('../computed/manifest-values.js');
 
 const UIStrings = {
   /** Title of a Lighthouse audit that provides detail on if a website is installable as an application. This descriptive title is shown to users when a webapp is installable. */
-  title: 'Web app manifest meets the installability requirements',
+  'title': 'Web app manifest and service worker meet the installability requirements',
   /** Title of a Lighthouse audit that provides detail on if a website is installable as an application. This descriptive title is shown to users when a webapp is not installable. */
-  failureTitle: 'Web app manifest does not meet the installability requirements',
+  'failureTitle': 'Web app manifest or service worker do not meet the installability requirements',
   /** Description of a Lighthouse audit that tells the user why installability is important for webapps. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
-  description: 'Browsers can proactively prompt users to add your app to their homescreen, ' +
-    'which can lead to higher engagement. ' +
+  'description': 'Service worker is the technology that enables your app to use many Progressive ' +
+    'Web App features, such as offline, add to homescreen, and push notifications. ' +
+    'With proper Service Worker and manifest implementations, browsers can proactively prompt ' +
+    'users to add your app to their homescreen, which can lead to higher engagement. ' +
     '[Learn more](https://web.dev/installable-manifest/).',
+  /** Description Table column header for the observed value of the Installability Error statistic. */
+  'columnValue': 'Installability Error',
```

Comment:
```suggestion
  'columnValue': 'Failure reason',
```

I have felt weird about these being called errors when it's not an error state, just a reason why it wouldn't be installed if one tried...

wdyt about listing these as "reasons" instead of "errors"

and `errorMessage` below can become `reason`

and probably a few other places like the displayValue

Reply:
yep this sgtm 👍